.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.idea
```

Comment:
What's this?

Reply:
jetbrains products generate a .idea folder in the project, added it to the gitignore so I don't add it accidentally